lib/mariaex/protocol.ex

```diff
@@ -76,6 +77,7 @@ defmodule Mariaex.Protocol do
     connect_opts = [host, opts[:port], opts[:socket_options], opts[:timeout]]
     binary_as    = opts[:binary_as] || :field_type_var_string
     datetime     = opts[:datetime] || :structs
+    json_library = opts[:json_library] || Poison
```

Comment:
Please allow setting the default value via `Application.get_env` so json library can be set in one place, with the default (`Poison`) set in the `mix.exs` `:env` entry.

Reply:
It fixed via f5703b2 .